VRDR/IJEMortality.cs

```diff
@@ -2617,27 +2482,62 @@ public string ACME_UC
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: AutomatedUnderlyingCauseOfDeath
-                return "";
+               return( ActualICD10toNCHSICD10(LeftJustified_Get("ACME_UC","AutoUnderlyingCOD")));
             }
             set
             {
-                // TODO: Implement mapping to FHIR record location: AutomatedUnderlyingCauseOfDeath
+                LeftJustified_Set("ACME_UC","AutoUnderlyingCOD", NCHSICD10toActualICD10(value));
             }
         }
 
         /// <summary>Entity-axis codes</summary>
+        // 20 codes, each taking up 8 characters:
+        // 1 char:   part/line number (1-6)
+        // 1 char: sequence within the line. (1-8)
+        // 4 char “ICD code” in NCHS format, without the ‘.’
+        // 1 char reserved”.  (not represented in the FHIR specification)
+        // 1 char ‘e code indicator’
         [IJEField(106, 715, 160, "Entity-axis codes", "EAC", 1)]
         public string EAC
         {
             get
             {
                 // TODO: Implement mapping from FHIR record location: EntityAxisCauseOfDeath
-                return "";
+                Tuple<string, string, string, string>[] eac = record.EntityAxisCauseOfDeath;
+                string eacStr = "";
+                foreach(Tuple<string, string, string, string> entry in eac)
+                {
+                    string lineNumber = entry.Item1;
```

Comment:
call Truncate and PadRight on each field to make sure they are the correct length